### ISSUE TITLE
Fails during postinstall script

### ISSUE THREAD

**Issue by Iverum** [appersonlabs/TiCalabash#30]
Trying to install ticalabash using `npm install -g ticalabash` results in everything going all right until the very end. I then get a message stating `Failed at the ticalabash@1.4.1 postinstall script 'node install'.`

```
npm install -g ticalabash
\
> ticalabash@1.4.1 postinstall /usr/local/lib/node_modules/ticalabash
> node install

Installing titanium stuff...
--- Installing Build Command(s) ---
ruby 2.0.0p481 (2014-05-08 revision 45883) [universal.x86_64-darwin14]
Password:
Successfully installed bundler-1.10.6
1 gem installed
Fetching gem metadata from https://rubygems.org/..........
Fetching version metadata from https://rubygems.org/..
Resolving dependencies...
Using CFPropertyList 2.3.1
Using awesome_print 1.6.1
Using builder 3.2.2
Using bundler 1.10.6
Using diff-lcs 1.2.5
Using multi_json 1.11.2
Using gherkin 2.12.2
Using multi_test 0.1.2
Using cucumber 1.3.20
Using escape 0.0.4
Using httpclient 2.6.0.1
Using json 1.8.3
Using retriable 1.4.1
Using rubyzip 1.1.7
Using slowhandcuke 0.0.3
Using calabash-android 0.5.12
Using calabash-common 0.0.1
Using edn 1.0.8 (was 1.0.7)
Using geocoder 1.2.9
Using thor 0.19.1
Using run_loop 1.4.0
Using rack 1.6.4
Using rack-protection 1.5.3
Using tilt 2.0.1
Using sinatra 1.4.6
Using sim_launcher 0.4.13
Using calabash-cucumber 0.14.3
Bundle updated!
events.js:85
      throw er; // Unhandled 'error' event
            ^
Error: spawn ti ENOENT
    at exports._errnoException (util.js:746:11)
    at Process.ChildProcess._handle.onexit (child_process.js:1053:32)
    at child_process.js:1144:20
    at process._tickCallback (node.js:355:11)
npm ERR! Darwin 14.4.0
npm ERR! argv "node" "/usr/local/bin/npm" "install" "-g" "ticalabash"
npm ERR! node v0.12.7
npm ERR! npm  v2.12.1
npm ERR! code ELIFECYCLE

npm ERR! ticalabash@1.4.1 postinstall: `node install`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the ticalabash@1.4.1 postinstall script 'node install'.
npm ERR! This is most likely a problem with the ticalabash package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node install
npm ERR! You can get their info via:
npm ERR!     npm owner ls ticalabash
npm ERR! There is likely additional logging output above.
```

**Comment by mguella** [comment]
+1

**Comment by mguella** [comment]
Got the same and solved with `npm install -g titanium`.
Problem was that the last exec of the postinstall (`ti config -a path.commands ...`) was searching for `ti`, but I only got `appc` and not anymore `ti` in my bash (new machine with latest appcelerator platform). Tried to alias `ti` to `appc ti` in the bash_profile, but didn't worked. So the only way for me was manually install the `titanium` package from npm.
